src/runtime/classobj.cpp

```diff
@@ -1516,6 +1525,7 @@ void setupClassobj() {
     instance_cls->giveAttr("__int__", new BoxedFunction(boxRTFunction((void*)instanceInt, UNKNOWN, 1)));
     instance_cls->giveAttr("__long__", new BoxedFunction(boxRTFunction((void*)instanceLong, UNKNOWN, 1)));
     instance_cls->giveAttr("__float__", new BoxedFunction(boxRTFunction((void*)instanceFloat, UNKNOWN, 1)));
+    instance_cls->giveAttr("__complex__", new BoxedFunction(boxRTFunction((void*)instanceComplex, UNKNOWN, 1)));
```

Comment:
Please see `test_complex`, line 228, there has class `class OS`, seems it could only work if I add `__complex__` to `classobj.cpp`.